gamemodes/terrortown/gamemode/client/cl_targetid.lua

```diff
@@ -275,6 +282,12 @@ function GM:HUDDrawTargetID()
 	-- only add onscreen infos when the entity isn't the local player
 	if ent == client then return end
 
+	local changedEnt = hook.Run("TTTModifyTargetedEntity", ent, distance)
```

Comment:
Useful getting an entity here instead of the modified data? I know it's easier to implement, but just thinking about a way to modify other things later, that could lead to incompatibilities.

Reply:
what do you mean by "modified data"?

Reply:
e.g. the text, etc. This current hook is just useful if you wanna switch the targeted player (which just makes sense for the disguiser, nothing else).

Reply:
the ext etc has already its own hook, `TTTRenderEntityInfo`. Besides, the entity has to be changed prior to the creation of the text etc. At this point, only the entity is available.

Reply:
So is it really needed or couldn't this be solved with the already existing hooks?

Reply:
The text can already be edited with existing hooks, the entity can't. Therefore a hook to edit the entity